ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSampler.scala

```diff
@@ -185,7 +179,9 @@ object BigSampler extends Command {
         args.optional("seed"),
         args.optional("distribution").map(SampleDistribution.fromString),
         args.list("distributionFields"),
-        Precision.fromBoolean(args.boolean("exact", default = false)))
+        Precision.fromBoolean(args.boolean("exact", default = false)),
+        args.optional("hashAlgorithm").map(HashAlgorithm.fromString).getOrElse(FarmHash)
```

Comment:
Nitpick: The order here should match the order used below where this comes after `field` and `seed`. That order is better since this field is directly related to the two preceding ones.

Reply:
Moved up.